client/src/App.js

```diff
@@ -4,14 +4,16 @@ import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
-
+import Navbar from "./components/Navbar";
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Route path="/" component={Navbar} />
+        {/* <Route path="/" component={LandingPage} /> */}
```

Comment:
Erase this commented code 😉

Reply:
Good catch, it's now removed and ready to go!